packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -913,6 +914,12 @@ class SemanticsProperties extends DiagnosticableTree {
   /// on iOS and TalkBack on Android).
   final SemanticsSortKey? sortKey;
 
+  /// Adds additional information onto the semantics subtree.
+  ///
+  /// This property provides a way for rendering objects to annotate their
+  /// semantics children without directly access them.
```

Comment:
Specifies a set of tags to be applied to the child SemanticsNodes of this widget.

The tag is added to all child SemanticsNodes that pass through the RenderObject corresponding to this widget while looking to be attached to a parent SemanticsNode.

Tags are used to communicate to a parent SemanticsNode that a child SemanticsNode was passed through a particular RenderObject. The parent can use this information to determine the shape of the semantics tree.

See also:

 * [SemanticsConfiguration.addTagForChildren], to which the tags provided here will be passed.